src/app/backend/resource/common/resourcechannels.go

```diff
@@ -160,7 +160,7 @@ func GetServiceListChannel(client client.Interface, nsQuery *NamespaceQuery,
 
 // IngressListChannel is a list and error channels to Ingresss.
 type IngressListChannel struct {
-	List  chan *extensions.IngressList
+	List  chan *networkingv1.IngressList
```

Comment:
Funny thing is that I think this is actually not used anywhere :joy: `ingress` package directly uses the `client.NetworkingV1` instead of channels. This is probably because it only requires a single call instead of multiple calls.

On the other hand `src/app/backend/client/manager.go` still creates a `k8sClient.ExtensionsV1beta1()` to create the `NewResourceVerber`. I guess this should also be updated.

Reply:
Hi @floreks 
According to https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/, `extensions/v1beta1` was only used for:

- `Ingress`, which this PR deals with
- `PodSecurityPolicy`, which is not yet handled by the dashboard
- `DaemonSet`, `Deployment` and `ReplicaSet` which are already handled by `apps/v1`

hence I just removed it's reference. This commit is trivial to commit, but please confirm you agree with the rationale behind it.

About CronJob (or any other thing that will be deprecated in the future, I'd rather wait to do the change. Not because of laziness, but because of backward compatibility. I know we only support officially the latest version, but I see no point in supporting a version that is not yet released ( 1.23 is less than a month old), since not everybody can upgrade k8s easily.